src/screens/validators/components/list/components/desktop/index.tsx

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
To follow our current folder structure we should make a `src/screens/validators/components/list/components/desktop/hooks.ts`

and move the `useEffect` in there

Reply:
Updated.